attributes/default.rb

```diff
@@ -27,6 +27,7 @@
 default['nodejs']['engine'] = 'node' # or iojs
 
 default['nodejs']['version'] = '0.10.26'
+default['nodejs']['use_latest_package'] = true
```

Comment:
I needed the ability to install different versions on different nodes, but I wanted to keep the previous behavior of installing the latest package intact for everyone else. Would prefer to remove the default version attribute and install latest if no version was given, but it would break the build_from_binary/source recipes without some additional work.